### PR TITLE
reimburse

### PR DESCRIPTION
**Pull request by hemangandhi**
So there's a new endpoint that directors can use to make the backend compute reimbursements.

Only users who could accept their HackRU RSVP (or didn't or whatever) can see their information - read is modified to this end.

### CODE REVIEW

File: reimburse.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        values = gmaps.distance_matrix(
+            origins = start_loc,
+            destinations = rac_address,
+            units = 'imperial',
+            mode = 'driving',
+            traffic_model = 'best_guess')
+
+    values *= config.TRAVEL.MULTIPLIERS[mode]
+    values = min(values, config.TRAVEL.MAX_REIMBURSE)
+
```
Comment (Sail338) [inline reimburse.py]: I hope this code works xD

File: reimburse.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    db = client['lcs-db']
+    db.authenticate(config.DB_USER,config.DB_PASS)
+    tests = db[config.DB_COLLECTIONS['users']]
+
+    gmaps = googlemaps.Client(key=config.MAPS_API_KEY)
+
+    for user in tests.find({"travelling_from": {"$exists": True}}):
+        do_one_user(user, gmaps)
+
+    return {'statusCode': 200, 'body': 'yote'}
```
Comment (Sail338) [inline reimburse.py]: yeet

File: reimburse.py [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+    client = MongoClient(config.DB_URI)
+    db = client['lcs-db']
+    db.authenticate(config.DB_USER,config.DB_PASS)
+    tests = db[config.DB_COLLECTIONS['users']]
+
+    gmaps = googlemaps.Client(key=config.MAPS_API_KEY)
+
+    for user in tests.find({"travelling_from": {"$exists": True}}):
+        do_one_user(user, gmaps)
```
Comment (Sail338) [inline reimburse.py]: I assume moving someone to 'coming' will add this field?

Comment (hemangandhi) [reply reimburse.py]: If they apply, it'll be there. The frontend adds it.

**Pull request merged**